src/riskmatrix/views/risk_assessment.py

```diff
@@ -154,14 +156,37 @@ def buttons(
         return assessment_buttons(assessment, self.request)
 
 
-_RADIO_TEMPLATE = Markup("""
+class AssessmentOverviewTable(AssessmentBaseTable):
+    nr = DataColumn(_("Nr."))
+    name = DataColumn(_("Name"))
+    description = DataColumn(_("Description"), class_name="visually-hidden")
+    category = DataColumn(_("Category"))
+    asset_name = DataColumn(_("Asset"))
+    likelihood = DataColumn(_("Likelihood"))
+    impact = DataColumn(_("Impact"))
+
+    def __init__(self, org: "Organization", request: "IRequest") -> None:
+        super().__init__(org, request, id="risks-table")
+        xhr_edit_js.need()
+
+    def query(self) -> "Generator[RiskMatrixAssessment]":
+        query = super().query()
+
+        for idx, item in enumerate(query):
+            item.nr = idx + 1
```

Comment:
```suggestion
        for idx, item in enumerate(query, start=1):
            item.nr = idx
```

Alright now I see what you use this `nr` for. This probably shouldn't really be stored on the model at all (definitely not inside the database through a `mapped_column`, a regular attribute should be fine). I would probably implement this using a subclass of `DataColumn`. It will be a little bit more tricky to get right though from that end, so maybe this should just be an option on the `DataTable` to prepend a numeric column, just like you can optionally add buttons, although I'm not sure how this will interact with custom sorting.

You could also potentially do it entirely through the frontend: https://datatables.net/examples/api/counter_columns.html